### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by BrandonSYu**
Display data map markers, new view-controller. Created annotation callout view displaying more details on foodtruck upon annotation tap. Pressing on this new callout view segues to advanced foodtruck details.

### CODE REVIEW

**Review by markify — COMMENTED** [review]
the map annotation call out works perfectly and displays the information.
Splash screen looks great!
We should have map loading when splash screen is loading to have map rendered faster for the user.
I added the segue map callout to the detail view controller which which works flawlessly.
Overall great job 😃

**Comment by BrandonSYu** [comment]
Good job making that segue from my annotation callout view, Mark! I'm going to try to work on a custom view for displaying multiple lines within my calloutview to display more details, such as operating hours, and address because right now it can only display one line.

Also, need to reduce clutter on map somehow to make the app more responsive.
Overall, great work team!

File: Find Food Trucks SF/Find Food Trucks SF/MapViewController.swift [hunk 1]
```diff
@@ -56,16 +58,41 @@ class MapViewController: UIViewController, CLLocationManagerDelegate {
...
-                
+                let annotation = FoodTruckAnnotation(title: foodTruck.name, subtitle: foodTruck.daysHours, coordinate)
                 self.mapView.addAnnotation(annotation)
             }
         }
     }
-    
 }
+
+extension MapViewController: MKMapViewDelegate {
```
Comment (markify) [inline Find Food Trucks SF/Find Food Trucks SF/MapViewController.swift:68]: In the map callout when marker is click we may be able to add the foodtruck address but so far it prints out the days and hours and food truck name.

**Review by markify — COMMENTED** [review]
Minor changes with map callouts

**Pull request merged**